tests/components/ring/test_camera.py

```diff
@@ -293,5 +344,145 @@ async def test_camera_image(
         "homeassistant.components.ring.camera.ffmpeg.async_get_image",
         return_value=SMALLEST_VALID_JPEG_BYTES,
     ):
-        image = await camera.async_get_image(hass, "camera.front")
+        image = await camera.async_get_image(hass, "camera.front_live_view")
         assert image.content == SMALLEST_VALID_JPEG_BYTES
+
+
+@pytest.mark.usefixtures("create_disabled_camera_entities")
+async def test_camera_stream_attributes(
+    hass: HomeAssistant,
+    mock_ring_client: Mock,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test stream attributes."""
+    await setup_platform(hass, Platform.CAMERA)
+
+    # Live view
+    state = hass.states.get("camera.front_live_view")
+    stream_type = state.attributes.get("frontend_stream_type")
+    assert stream_type is camera.StreamType.WEB_RTC
```

Comment:
The frontend stream type is deprecated and should not be used anymore.
Please use camera capabilities instead

Reply:
Done